ProCharity

lucy2n opened the pull request.

a-kina00 committed:
style: правки в файловой структуре

a-kina00 committed:
fix: убрано добавление шрифта

Commit:
feat: добавлен статус провиля и его заполнение

Commit:
fix: исправила transition на кнопке и отступы в title

Commit:
feat: структура footer

Kukus89 committed:
навтроить инпут с выпадающим меню средствами targify

Falor89 committed:
feat: добавил фундамент первого попапа

Kukus89 committed:
Исправить инпуты, добавить выпадающее меню, отредактировать инпут в соответствии с фигмой, убрать лишние обёртки у инпутов, удалить лишний код.

Kukus89 committed:
fix: Удалить фон у кнопки "назад"

Kukus89 committed:
fix: исправление по ревью, + у инпута добавлен текст-оверфлоу, убрано наложение текста на значки.

Commit:
feat: адаптированно для мобильной версии

Commit:
feat: адаптация для мобильной версии

lucy2n committed:
Merge pull request #6 from lucy2n/features/inputs&buttons

Features/inputs&buttons

Commit:
feat: цвет footer

lucy2n committed:
Merge branch 'develop' into features/header

Commit:
feat: фон

lucy2n committed:
fix: исправить мерж конфликты

lucy2n committed:
Merge pull request #1 from lucy2n/features/header

Features/header

Commit:
feat: почти реализована полная версия

lucy2n committed:
Merge branch 'develop' into features/popups

# Conflicts:
#	UIKit.html
#	src/pages/index.css

lucy2n committed:
feat: добавить cropperjs

a-kina00 committed:
feat: добавлена мобильная вёрстка

a-kina00 committed:
fix: исправлены цвета и названия

a-kina00 committed:
Merge branch 'develop' into features/tags

# Conflicts:
#	UIKit.html
#	src/pages/index.css

a-kina00 committed:
fix: добавлен html тегов

lucy2n committed:
feat: настройка cropperjs

a-kina00 committed:
fix: мёрдж  'develop' в features/menu

# Conflicts:
#	UIKit.html
#	src/pages/index.css

a-kina00 committed:
fix: добавлен html меню

lucy2n committed:
Merge pull request #2 from lucy2n/features/tags

feat: вёрстка тегов

a-kina00 committed:
fix: мёрдж branch 'develop' в features/menu

# Conflicts:
#	src/UIKit.html
#	src/pages/index.css

a-kina00 committed:
fix: исправлено название модификатора

lucy2n committed:
Merge pull request #3 from lucy2n/features/menu

Features/menu

lucy2n committed:
fix: исправить имя класса

lucy2n committed:
feat: мердж develop в feature/status

lucy2n committed:
Merge pull request #5 from lucy2n/features/status

feat: добавлен статус провиля и его заполнение

Falor89 committed:
feat: обновление попап-функциональности

Falor89 committed:
feat: Доделан попап-фотография

Kukus89 committed:
Создание структуры страницы контактов

Kukus89 committed:
Размещены элементы страницы

Kukus89 committed:
правка цсс

KlyukovaK committed:
feat: footer десктопная версия

Falor89 committed:
feat: попап-модерация

Falor89 committed:
feat: попап-уведомление доделан

Kukus89 committed:
Подправил инпуты

Falor89 committed:
fix: исправление отступов и импутов

KlyukovaK committed:
feat: реализована мобильная версия

lucy2n committed:
fix: добавить aspectRatio для cropper

lucy2n committed:
fix: добавлен модификатор

lucy2n committed:
Merge branch 'develop' into features/popups

lucy2n committed:
Merge pull request #9 from lucy2n/features/popups

Features/popups

Commit:
feat: переделана кнопка под личный кабинет

KlyukovaK committed:
fix: исправила отступ

Kukus89 committed:
Контрольная точка

Falor89 committed:
feat: добавил UiKit

Commit:
feat: обозначина структура страницы

Commit:
fix: убрала отступы слева у profress & status, так как в общей верстке они не нужны

lucy2n committed:
Merge pull request #11 from lucy2n/fix/status

fix: убрала отступы слева у profress & status, так как в общей верстк…

lucy2n committed:
Merge pull request #10 from lucy2n/fix/header

feat: переделана кнопка под личный кабинет

lucy2n committed:
Merge branch 'develop' into features/footer

lucy2n committed:
Merge pull request #7 from lucy2n/features/footer

Features/footer

Kukus89 committed:
Сверстать мобильную версию

Commit:
feat: почти реализована десктопная версия

Kukus89 committed:
Сделать моб.версию, переделать шаблон на гриды

Kukus89 committed:
сделать на гридах, отступы под фигму, доводка

Kukus89 committed:
fix: fix всплывающей подсказки

Kukus89 committed:
fix: в подвале заменил ссылки img на "вебкитовские"

Commit:
feat: реализована мобилдьная версия

Kukus89 committed:
Контрольная точка

lucy2n committed:
Merge branch 'develop' into features/v_account/motivation

lucy2n committed:
Merge pull request #12 from lucy2n/features/v_account/motivation

Features/v account/motivation

Commit:
fix: добавлен footer

Kukus89 committed:
Исправления по ревью

Kukus89 committed:
fix: фикс кнопки хедера

Commit:
feat: структура страницы profile

Commit:
fix: UIKit footer

Kukus89 committed:
fix: ренейм блоков в соответствии в вложенностью

lucy2n committed:
Merge pull request #13 from lucy2n/fix/v_account/motivation

fix: добавлен footer

Kukus89 committed:
контрольная точка перед пулл-рек.

lucy2n committed:
fix: изменены цвета page и content, подредактировала index.css, переместила картинку

lucy2n committed:
Merge branch 'develop' into fix/styles

Kukus89 committed:
fix: создан отдельный цсс для секции "contacts"

Falor89 committed:
feat: добавил чекбоксы

Kukus89 committed:
Merge remote-tracking branch 'origin/develop' into feat/v_account/contacts

a-kina00 committed:
fix: исправлено отображение меню на разной ширине

lucy2n committed:
Merge pull request #15 from lucy2n/features/menu

fix: исправлено отображение меню на разной ширине

Kukus89 committed:
создать ветку social

Kukus89 committed:
Merge remote-tracking branch 'origin/develop' into feat/v_account/contacts

Kukus89 committed:
fix: модификатор для contents

Kukus89 committed:
Стартовая разметка секции

Kukus89 committed:
Merge branch 'feat/v_account/contacts' into feat/v_account/social

Commit:
fix: разметка страницы

Commit:
fix: css content

Commit:
fix: css content

Falor89 committed:
feat: add telegram div

Commit:
feat: реализована страница profile

Kukus89 committed:
fix:  доводка шаблона

Falor89 committed:
feat: доделана вся страничка, кроме попапа

Falor89 committed:
feat: начал доделывать уведомление

lucy2n committed:
Merge branch 'develop' into fix/v_account/motivation

lucy2n committed:
Merge pull request #16 from lucy2n/fix/v_account/motivation

fix: разметка страницы

lucy2n committed:
Merge branch 'develop' into features/v_account/profile

lucy2n committed:
Merge pull request #17 from lucy2n/features/v_account/profile

Features/v account/profile

Kukus89 committed:
fix: унификация мейна

Kukus89 committed:
унификация мейна

Falor89 committed:
fix:small fix

Kukus89 committed:
"уборка"  в файловой структуре страницы

Kukus89 committed:
унификация мейна

Klyukova committed:
feat: добавлен общий класс content и main

Kukus89 committed:
fix: правка инпутов (крестик)

lucy2n committed:
Merge pull request #18 from lucy2n/fix/motivation/profile

feat: добавлен общий класс content и main

Kukus89 committed:
Merge remote-tracking branch 'origin/fix/motivation/profile' into feat/v_account/social

Kukus89 committed:
Контрольная точка

Kukus89 committed:
Merge remote-tracking branch 'origin/develop' into feat/v_account/social

Kukus89 committed:
ренейм классов, унификация мейна

Kukus89 committed:
убрать лишний класс на кнопке

Kukus89 committed:
фикс кнопки

Kukus89 committed:
Merge branch 'feat/v_account/social' into feat/v_account/contacts

Kukus89 committed:
Унификация мейна, ренейм классов

Kukus89 committed:
правка инпутов

Klyukova committed:
feat: реализация password

Kukus89 committed:
контрольная точка

lucy2n committed:
feat: базовая имплементация многоуровнего дропдауна, обновить .gitignore

a-kina00 committed:
fix: исправлены теги меню

Kukus89 committed:
Контрольная точка перед отправкой

a-kina00 committed:
feat: мёрдж 'develop' в features/menu

# Conflicts:
#	src/UIKit.html

a-kina00 committed:
fix: исправлено поведение меню регистрации при наведении

lucy2n committed:
Merge pull request #21 from lucy2n/features/menu

Features/menu

a-kina00 committed:
fix: исправлена ширина тегов

lucy2n committed:
Merge pull request #20 from lucy2n/features/v_account/password

feat: реализация password

lucy2n committed:
Merge pull request #22 from lucy2n/features/tags

fix: исправлена ширина тегов

Kukus89 committed:
правки наименований

lucy2n committed:
feat: улучшить стили многоуровнего списка

lucy2n committed:
feat: улучшить стили многоуровнего списка

lucy2n committed:
fix: разнести js по разным файлам, подправить стили

lucy2n committed:
Merge branch 'develop' into features/dropdown-multilevel

a-kina00 committed:
feat: добавлена страница регистрации

lucy2n committed:
fix: подправила css

lucy2n committed:
Merge branch 'features/dropdown-multilevel' of github.com:lucy2n/ProCharity-project into features/dropdown-multilevel

lucy2n committed:
fix: убран scroll bar в dropdown

lucy2n committed:
Merge pull request #23 from lucy2n/features/dropdown-multilevel

Multilevel dropdown

a-kina00 committed:
feat: добавлена страница с email

a-kina00 committed:
fix: мёрдж 'develop' в registration/check_e-mail

# Conflicts:
#	src/pages/index.css
#	src/pages/index.js
#	webpack.config.js

Klyukova committed:
feat: реализация страницы competition без popup

lucy2n committed:
fix: исправленны стили дропдауна

lucy2n committed:
Merge branch 'features/dropdown-multilevel' into develop

Klyukova committed:
feat: создан popup

Falor89 committed:
feat: обновил чекбоксы и мобильную версию

lucy2n committed:
Merge pull request #25 from lucy2n/features/v_account/competition

Features/v account/competition

Falor89 committed:
fix: подтянул дев, изменил кнопку телеграм

lucy2n committed:
fix: исправленны стили дропдауна, добавленна обводка

lucy2n committed:
Merge branch 'features/dropdown-multilevel' into develop

lucy2n committed:
fix: исправленны стили дропдауна

lucy2n committed:
Merge branch 'features/dropdown-multilevel' into develop

Falor89 committed:
fix: переписал на main

Klyukova committed:
feat: разметка страницы portfolio

a-kina00 committed:
feat: добавлена страница регистрации волонтёра

a-kina00 committed:
fix: мёрдж 'develop' в feat/registration/volunteer

# Conflicts:
#	src/pages/index.css
#	webpack.config.js

Falor89 committed:
fix: изменение кнопки личного кабинета

lucy2n committed:
Merge branch 'develop' into feat/v_account/notifications

lucy2n committed:
Merge pull request #29 from lucy2n/feat/v_account/notifications

Feat/v account/notifications

Kukus89 committed:
Исправлен цвет текста в инпуте

Kukus89 committed:
Fix: инпуты

Kukus89 committed:
фикс: подправил цвета инпутов

Klyukova committed:
feat:реализация страницы portfolio

lucy2n committed:
feat: добавлен стиль одноуровнего инпута

lucy2n committed:
Merge branch 'feat/v_account/contacts' into develop

# Conflicts:
#	src/UIKit.html
#	src/blocks/input-case/__dropdown-content/input-case__dropdown-content.css
#	src/pages/index.css
#	src/v_account_motivation.html
#	src/v_account_profile.html
#	webpack.config.js

lucy2n committed:
Merge pull request #19 from lucy2n/feat/v_account/social

Feat/v account/social

lucy2n committed:
Merge branch 'develop' into features/v_account/portfolio

lucy2n committed:
Merge pull request #30 from lucy2n/features/v_account/portfolio

Features/v account/portfolio

lucy2n committed:
fix: добавить page__content

a-kina00 committed:
fix: исправлены позиции элементов

a-kina00 committed:
fix: мёрдж 'develop' в registration/check_e-mail

# Conflicts:
#	src/pages/index.css
#	webpack.config.js

a-kina00 committed:
fix: мёрдж 'develop' в registration/create-account

# Conflicts:
#	src/pages/index.css
#	src/pages/index.js
#	webpack.config.js

a-kina00 committed:
fix: мёрдж 'develop' в feat/registration/volunteer

# Conflicts:
#	src/blocks/input/input.css
#	src/pages/index.css
#	webpack.config.js

a-kina00 committed:
fix: изменения в инпутах

Klyukova committed:
fix: исправила инпут для текста

lucy2n committed:
Merge pull request #32 from lucy2n/fix/all/pages

fix: исправила инпут для текста

lucy2n committed:
fix: исправить notification__mark

lucy2n committed:
Merge branch 'develop' into registration/check_e-mail

lucy2n committed:
Merge pull request #24 from lucy2n/registration/check_e-mail

Registration/check e mail

Kukus89 committed:
Fix: исправление вложенности, переименовывание классов

a-kina00 committed:
fix: исправлено расположение элементов

a-kina00 committed:
fix: мёрдж 'develop' в feat/registration/volunteer

# Conflicts:
#	src/pages/index.js
#	webpack.config.js

a-kina00 committed:
fix: исправлено расположение файлов

a-kina00 committed:
fix: мёрдж 'develop' в registration/create-account

# Conflicts:
#	src/blocks/link/link.css
#	src/pages/index.js
#	webpack.config.js

lucy2n committed:
Merge pull request #31 from lucy2n/registration/create-account

Registration/create account

lucy2n committed:
Merge branch 'develop' into fix/contacts-class-name

lucy2n committed:
Merge pull request #33 from lucy2n/fix/contacts-class-name

Fix: исправление вложенности, ренейм классов

a-kina00 committed:
fix: исправлены блоки под шаблон

a-kina00 committed:
Merge branch 'develop' into feat/registration/volunteer

# Conflicts:
#	webpack.config.js

a-kina00 committed:
fix: исправлена страница регистрации волонтёра

lucy2n committed:
fix: подправить стили

lucy2n committed:
Merge pull request #28 from lucy2n/feat/registration/volunteer

Feat/registration/volunteer

lucy2n committed:
fix: исправить наименование некоторых стилей

lucy2n committed:
fix: добавить стили

lucy2n committed:
fix: вернуть утеряную страницу socila

lucy2n committed:
fix: исправленны проблемы с валидацией html

lucy2n committed:
fix: изменена структура проекта

lucy2n committed:
fix: исправление стилей

lucy2n committed:
feat: добавлен README

lucy2n committed:
feat: обновить README

lucy2n committed:
fix: исправление стилей из ветки fix/popup

lucy2n committed:
fix: исправление стилей popup__notification

lucy2n committed:
fix: подправлено форматирование кода

lucy2n committed:
feat: обновить README

lucy2n committed:
fix: обновить README

ilanabatavraam reviewed (commented):
Здравствуйте! Спасибо за вашу работу! Ниже вы можете ознакомиться с критериями, по которым она была оценена:


## Критерии

1. Соответствие макету:
    - [ ] <!--12.0--> 1.1 Все элементы из макета сверстаны // попап "Фото" не открывается
    - [x] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету
    - [x] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета
    - [x] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser
2. Адаптив:
    - [x] <!--4.0--> 2.1 На всех размерах экрафна 320px и более отсутствует горизонтальный скролл
    - [ ] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету // https://skr.sh/sJPl1sZMez0
    - [x] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами
    - [ ] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один // https://skr.sh/sJP8FDUTdUU
    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более 
    - [x] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
    - [ ] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются // дублируются в `.create-account__title`, `.create-account__text`, `.navigation`, `.registration__title`
3. Верстка:
    - [x] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)
    - [ ] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо  // логотип на внутренних страницах должен быть ссылкой на главную // некорректные значения `type="text"` для полей для ссылок https://skr.sh/sJPM4o6jcsE // `.button__type_telegram` должен быть ссылкой
    - [ ] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>` // https://skr.sh/sJP15yoSNKL
    - [ ] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки // все `div.main__checkbox.main__checkbox_notification`
    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [ ] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы // на некоторых страницах пропущены уровни заголовков. например, после h1 сразу используется h3. можно быстро проверить через сервис https://yoksel.github.io/html-tree/
    - [ ] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`. // необходимо на всех страницах заполнить актуальный title в едином стиле (на одном языке)
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [ ] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении // `status`, `.popup__container`, `.popup__div`, `.popup__notification`
    - [x] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность
    - [ ] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта // https://skr.sh/sJPLrUfAzcH https://skr.sh/sJPxzvKv3wS https://skr.sh/sJPiLnn1Xu2
5. Применение БЭМ:
    - [ ] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют // модификатор `.button_type_disabled`, `.radio-box__button_type_active` по смыслу булевый, поэтому нужно использовать простой булевый модификатор вместо сложного ключ-значение // `button_small`, `.button__type_telegram`, `.content_config-contacts`, `.create-account_left`, `.create-account_right`, `.main__checkbox_notification`, `.main__files_add `, `.notification__image_question` некорректное использование булевого модификатора, его необходимо заменить на сложный ключ-значение + ошибки (нужно проверить все страницы) нужно поймать через сервис https://yoksel.github.io/html-tree/
    - [ ] <!--2.0--> 5.2 Не используются селекторы по тегу // стилизовать необходимо через класс, не через тег https://skr.sh/sJPfp1OvBsb
    - [ ] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока // `.competition-popup` и `.popup`
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [x] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)
    - [ ] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства // содержат модификаторы от `.create-account` (часть стилей можно вынести в основной блок)
    - [ ] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested // селектор должен стилизоваться в своем css файле https://skr.sh/sJP0CkDJ9tj https://skr.sh/sJPMMifj2rc https://skr.sh/sJPg1s2i7cE https://skr.sh/sJPNbr6dSiu https://skr.sh/sJPTLxcSJR1 https://skr.sh/sJPouViHQcx https://skr.sh/sJP6K6vLYHz https://skr.sh/sJPf5IFWV2t https://skr.sh/sJPxodqZbtq https://skr.sh/sJP1eZXa2rq https://skr.sh/sJPrtfW4I8C https://skr.sh/sJP6m36xww8 https://skr.sh/sJPn0jsWPMY https://skr.sh/sJP3MVeOnEg https://skr.sh/sJPQUcVClyJ https://skr.sh/sJP6YnNcVyL https://skr.sh/sJPnefu4gA7 // некорректное название папки модификатора `src/blocks/input-case/__error-message/_type/input-case-error-message_type-active.css`, `src/blocks/main/__image-container/__type` // некорректное название css файла `src/blocks/input-case/__image/_active/input-case__image__active.css`, `src/blocks/radio-box/__button/_type-active` // ошибка в названии класса https://skr.sh/sJPBRta4Vvk // папку для анимаций `keyframes/` и объявление переменных `:root` необходимо вынести из `blocks/` // некорректное название папки блока `src/blocks/position/fixed.css`
- [ ] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект) // см. "Доп. комментарии"
7. Структура проекта и оформление кода
    - [x] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта
    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов

## Количество баллов: 49

## Доп. комментарии:

Не выполнено:

- Не работает переход между страницами (при сохранении формы)
- Страница “Личные данные”: 
  - При первом заполнении данных после нажатия кнопки "сохранить" система сохраняет данные и переводит пользователя в следующий подраздел “Мои компетенции” для заполнения профиля. 
  - Поля в блоке “фонды увидят эти данные на твоей странице” и в блоке “Контактные данные” - обязательны к заполнению.
  - В поле “предпочтительный способ связи” выпадает список из позиций: электронная почта, телефон, телеграмм. Поля подтягиваются из Битрикса. Выбор не работает.
  - При нажатии на кнопку “х” удаляется весь введенный текст.
  - Не получается выбрать пункт "я представляю компанию"

- Страница “Мои компетенции”
  - на мобильном/планшете нет возможности свернуть текущую вкладку и выбрать/сбросить любой пункт https://disk.yandex.ru/i/LPcCncblIFcHDw

- Страница ”Моя мотивация и сильные стороны”
  - Поле автоматически расширяется при вводе текста. Появляется скролл, но поле не расширяется https://skr.sh/sJPdaKRofWe
  - При вводе текста необходимо обновлять счетчик символов (стиль и положение описаны макетом в figma), т.к. в textarea по-дефолту нет счетчика символов.
  - При нажатии на кнопку “х” удаляется весь введенный текст.

- Страница “Мое портфолио”
  - При нажатии на значек “корзина” документ удалятся, нижние поля с документами смещаются вверх.

- Страница “Регистрация (создание аккаунта)”
  - После нажатия на кнопку должна открыться страница “Регистрация нового волонтера” / “Регистрация нового   -   - После нажатия на слово “правилами регистрации … ” должна открыться ссылка в новом окне, ссылка пока пустая.
  - После нажатия на фразу “условиях ….” должна открыться ссылка в новом окне, ссылка пока пустая.
  - При клике на слово “Войти” должна открыться страница “Авторизация” - данная страница существует, верстать ее не нужно.

- Страница “Регистрация нового волонтера”
  - Кнопка “зарегистрироваться” изначально неактивна. Кнопка активируется только после того как будут заполнены все поля и поставлена галочка в чек-боксе.
  - После нажатия на кнопку “зарегистрироваться” происходит валидация данных.
  - Кнопка “назад” всегда активна.
  - Ссылка “правилами сайта” кликабельна. После нажатия на слово “правилами сайта” должна открыться ссылка в новом окне, ссылка пока пустая. При возврате с новой ссылки в окно регистрации введенные пользователем данные НЕ сохраняются в памяти.
  - После нажатия на кнопку “зарегистрироваться” открывается страница “первый шаг к добрым делам”. Также на указанную почту должно отправиться письмо с уведомлением о регистрации.

lucy2n committed:
fix: пункт 2.4

lucy2n committed:
fix: пункт 2.7

lucy2n committed:
fix: пункт 3.8

lucy2n committed:
fix: пункт 4.5

lucy2n committed:
fix: пункт 5.2

lucy2n committed:
fix: пункт 3.7

lucy2n committed:
fix: исправить cropper

lucy2n committed:
fix: пункт 3.2

lucy2n committed:
feat: добавить открытие и закрытие попапа

lucy2n committed:
fix: пункт 3.3

lucy2n committed:
feat: добавлен js для dropdown input

lucy2n committed:
fix: исправить модификатор

Kukus89 committed:
создать функцию очистки инпутов

lucy2n committed:
fix: исправить проблему с фокусом dropdown

Kukus89 committed:
добавил очистку импутов

lucy2n committed:
fix: пункт 4.3

lucy2n committed:
fix: исправить main__paragraph

lucy2n committed:
fix: добавить закрытие popup__notification

lucy2n committed:
fix: добавить закрытие competition-popup

Kukus89 committed:
исправлены инпуты на страницах, добавлен импорт функции очистки

Kukus89 committed:
Контрольная точка, правка отступов

Klyukova committed:
feat: счетчик в инпуте

Kukus89 committed:
Исправлен "крестик" для textarea

lucy2n committed:
feat: добавить функционал radio_button

lucy2n committed:
Merge pull request #37 from lucy2n/fix/counter

feat: счетчик в инпуте

lucy2n committed:
Merge branch 'fix/pr' into fix/inputClear

lucy2n committed:
Merge pull request #36 from lucy2n/fix/inputClear

Fix/input clear

lucy2n committed:
fix: исправить input

lucy2n committed:
fix: исправить импорты

a-kina00 committed:
fix: исправлена максимальная ширина страницы

lucy2n committed:
fix: пункт 5.7

a-kina00 committed:
fix: merge 'fix/pr' into fix/create-account

# Conflicts:
#	src/blocks/create-account/__background/create-account__background.css

a-kina00 committed:
fix: исправлено выравнивание

lucy2n committed:
fix: исправлены отступы у блока

lucy2n committed:
Merge pull request #38 from lucy2n/fix/create-account

fix: исправлена максимальная ширина страницы

lucy2n committed:
fix: исправление стилей

lucy2n committed:
fix: исправление стилей

lucy2n committed:
fix: исправление стилей

lucy2n committed:
feat: добавить функционал сокрытия пароля

lucy2n committed:
feat: добавить открытие competition-popup

lucy2n committed:
fix: исправление стилей main__notification-mark

Klyukova committed:
feat: обнуление счетчика

lucy2n committed:
fix: исправление стилей

lucy2n committed:
Merge remote-tracking branch 'origin/fix/deleteCounter' into fix/pr

ilanabatavraam reviewed (commented):
Отличная работа!🔥

Ваша команда отлично справилась с заданием 🚀 Поздравляю каждого участника!

## Критерии

1. Соответствие макету:
    - [x] <!--12.0--> 1.1 Все элементы из макета сверстаны
    - [x] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету
    - [x] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета
    - [x] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser
2. Адаптив:
    - [x] <!--4.0--> 2.1 На всех размерах экрафна 320px и более отсутствует горизонтальный скролл
    - [x] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету
    - [x] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами
    - [x] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один
    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более 
    - [x] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
    - [x] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются
3. Верстка:
    - [x] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)
    - [x] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо
    - [x] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>`
    - [x] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки 
    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [x] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы
    - [x] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`.
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [x] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении
    - [x] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность
    - [x] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта
5. Применение БЭМ:
    - [x] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют
    - [x] <!--2.0--> 5.2 Не используются селекторы по тегу
    - [ ] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока // `.competition-popup` и `.popup`
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [x] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)
    - [x] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства
    - [x] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested
- [ ] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект) // см. "Доп. комментарии"
7. Структура проекта и оформление кода
    - [x] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта
    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов


## Количество баллов: 87

## Доп. комментарии:

- Страница “Мои компетенции”
  - на мобильном/планшете нет возможности свернуть текущую вкладку и выбрать/сбросить любой пункт

- Страница ”Моя мотивация и сильные стороны”
  - Поле автоматически расширяется при вводе текста. Появляется скролл, но поле не расширяется

- Страница “Мое портфолио”
  - При нажатии на значек “корзина” документ удалятся, нижние поля с документами смещаются вверх.

- Страница “Регистрация (создание аккаунта)”
  - После нажатия на слово “правилами регистрации … ” должна открыться ссылка в новом окне, ссылка пока пустая.
  - После нажатия на фразу “условиях ….” должна открыться ссылка в новом окне, ссылка пока пустая.

- Страница “Регистрация нового волонтера”
  - Ссылка “правилами сайта” кликабельна. После нажатия на слово “правилами сайта” должна открыться ссылка в новом окне, ссылка пока пустая.
